Bomberfrau/src/main/java/uni/bombenstimmung/de/game/Field.java

```diff
@@ -109,13 +110,17 @@ public void drawField(Graphics g, int x, int y, FieldContent cont) {
 		} 	else if (Game.getMapNumber() == 2) {
 		    	g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_YELLOWGRAS).getImage(), x, y, GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
 		}	else if (Game.getMapNumber() == 3) {
-					g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_LAVA_FLOOR).getImage(), x, y, GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
-				}
-		g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_BOMB).getImage(), x, y, GameData.FIELD_DIMENSION-5, GameData.FIELD_DIMENSION-5, null);
-		for(Bomb i : Game.getPlacedBombs()) {
-		    i.drawCounter(g);
-		    //int count = i.getCounter();
-		    //GraphicsHandler.drawCentralisedText(g, Color.RED, 30, i.getCounter() + "", i.getPlacedField().xPosition+(GameData.FIELD_DIMENSION/2), i.getPlacedField().yPosition+(GameData.FIELD_DIMENSION/2));
+		    g.drawImage(ImageHandler.getImage(ImageType.IMAGE_INGAME_LAVA_FLOOR).getImage(), x, y, GameData.FIELD_DIMENSION, GameData.FIELD_DIMENSION, null);
+		}
+		try {
+		    for(Bomb i : Game.getPlacedBombs()) {
```

Comment:
`for(Bomb i : Game.getPlacedBombs()) {`
Bitte Variable passend bennen. 
Z.B. 
`for(Bomb bomb : Game.getPlacedBombs()) {`